readme-docs/archive/readme-v3.1.2-release.md

```diff
@@ -0,0 +1,63 @@
+# Frames Android SDK
+[![CircleCI](https://circleci.com/gh/checkout/frames-android/tree/master.svg?style=svg)](https://circleci.com/gh/checkout/frames-android/tree/master)
+[![](https://jitpack.io/v/checkout/frames-android.svg)](https://jitpack.io/#checkout/frames-android)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
+
+Start accepting online payments in just a few minutes with our Android SDK. It's quick and easy to integrate, accepts online payments from all major credit cards, and is customizable to your brand.
+
+## Features
+
+#### Two ways to integrate:
+- [Use our native UI](https://www.checkout.com/docs/integrate/sdks/android-sdk#Usage_(with_the_module's_UI)) and embed the fully customisable form to accept card details, customer name and billing details and exchange them for a secure token.
+- [Build your own UI](https://www.checkout.com/docs/integrate/sdks/android-sdk#Usage_(without_the_module's_UI)) and use the provided API to send sensitive data to the Checkout.com server and retrieve the secure token.
+
+**Accept cards and Google Pay** easily using our UI and tokenization module to stay away from PCI-compliance.
```

Comment:
Through the PR we use tokenises with `z` and `s` we can align to ios with tokenises with s.

Reply:
No need to review the previous readme file in archive.
archive/readme-v3.1.2-release.md